src/styles/main.scss

```
@@ -1,3 +1,74 @@
+$Day-box-width: 100px;
+$Day-box-height: 100px;
+$Main-color: black;
+$Day-box-border: 1px;
+$start-day: 0;
+
 body {
+  height: 100vh;
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+  padding: 10px;
+
+  $days:
+    "tue" 101px,
+    "wed" 202px,
+    "thu" 303px,
+    "fri" 404px,
+    "sat" 505px,
+    "sun" 606px;
```

Don't set such a values. Instead of pixels just put the order of days:
```suggestion
  $days:
    "mon" 1,
    "tue" 2,
    "wed" 3,
// etc...
```